Sources/Kitura/Kitura.swift

```diff
@@ -105,6 +105,7 @@ public class Kitura {
     // MARK: Start Servers
     
     /// Start the Kitura framework.
+    /// Can also provide a value for the withStatus param, this will enable the process to exit with an exit code should any of the servers fail to start.
```

Comment:
This sentence could do with rewording.
Plus you need to add the `withStatus` parameter and its description to the jazzy doc.

Reply:
Reworded to what?
I'll look into how to add the `withStatus` parameter to the Jazzy docs, it's been a long time since I had to work with Jazzy. 

Can we add a test for this? Do the Kitura tests allow for us to spin up multiple servers on the same port within within the same test?

Reply:
Sorry David, more info on wording... I'd expect the parameter doc for `withStatus` to say something like:

`withStatus` "Determines whether the Kitura process can return a non-zero exit code should any of the servers fail to start. Defaults to false, indicating it will not return any non-zero exit codes."

I'd expect the descriptive wording to be something like.
/// Start the Kitura framework.
/// By default, the Kitura framework process will exit with a code of zero, even if one or more of the servers fails to start.  To enable the Kitura framework process to exit with a non-zero exit code set the `withStatus` parameter to true.

Reply:
Thanks for providing that, it definitely reads better than what I had :). 

I've just pushed up your suggested changes.